### PR TITLE
Rename ApiKey#id; change LogController return type

### PR DESCRIPTION
**Pull request by landonreed**
### Checklist

- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [ ] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [ ] The description lists all applicable issues this PR seeks to resolve
- [ ] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

This PR 
- changes `ApiKey#id` to `ApiKey#keyId` (Mongo doesn't seem to like queries that try to reference `id`) and
- modifies the return type of LogController so that ApiUsers can be more easily associated with the usage for different key ids.

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/models/ApiUsageResult.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.middleware.models;
+
+import com.amazonaws.services.apigateway.model.GetUsageResult;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ApiUsageResult implements Serializable {
```
Comment (br648) [inline src/main/java/org/opentripplanner/middleware/models/ApiUsageResult.java:16]: JavaDoc

File: src/main/java/org/opentripplanner/middleware/models/ApiUsageResult.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public final GetUsageResult result;
+    public final Map<String, ApiUser> apiUsers;
+
+    public ApiUsageResult(GetUsageResult result) {
```
Comment (br648) [inline src/main/java/org/opentripplanner/middleware/models/ApiUsageResult.java:24]: JavaDoc.

**Review by br648 — APPROVED** [review]
LGTM, just a couple of Javadoc additions needed.

**Review by evansiroky — APPROVED** [review]
I second Robin's request for just a little more JavaDoc

**Pull request merged**